tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "tmp",
         "temp",
         "build",
-        "node_modules"
+        "node_modules",
+        "test/utils/test-runner.ts"

```

Comment:
Why this file is excluded and runned with ts-node?

Reply:
If that file is included in tranpilation, no tests are executed. I think it's some issue with the mocha global. I had started to run tests using ts-lint entirely but stopped after having some problems with Travis. Probably a good idea to take that up again and see if it will work. It should make local runs easier and allow us to exclude all tests from the build, which will make things faster.